plugins/kubernetes/app/controllers/kubernetes_roles_controller.rb

```diff
@@ -21,19 +21,12 @@ def update
     end
   end
 
-  def new
-    # TODO : READ DEFAULT VALUES FROM YAML FILE
-    render json: Kubernetes::Role.new(project: current_project, ram: 512, cpu: 0.2, replicas: 1, deploy_strategy: 'rolling_update'), root: false
-  end
-
-  def create
-    role = current_project.roles.build(role_params)
-    role.save
-
-    if role.persisted?
-      render status: :created, json: role
+  def refresh
+    roles = current_project.refresh_kubernetes_roles!(refresh_params)
+    if roles.try(:any?)
```

Comment:
Seems a bit odd to enumerate the whole list here to check if it's empty? Why not just ```roles.empty?```